### PR TITLE
add BasicItem mod and beginning of AdvancedItem mod

### PR DESCRIPTION
**Pull request by VeryGoodDog**

### CODE REVIEW

File: mods/AdvancedItem/assets/advanceditem/recipes/grid/wand.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"ingredientPattern": "P,S,S",
+	"ingredients": {
+		
+	}
```
Comment (p3t3rix) [inline mods/AdvancedItem/assets/advanceditem/recipes/grid/wand.json:5]: seems like you're missing the ingredients, is this intended ?

Comment (VeryGoodDog) [reply mods/AdvancedItem/assets/advanceditem/recipes/grid/wand.json:5]: yeah that mod was only started, i probably should not have committed it.

**Comment by tyronx** [comment]
Neat, but if you say the beginning of xy mod - is it ready to pull then?

**Comment by VeryGoodDog** [comment]
> Neat, but if you say the beginning of xy mod - is it ready to pull then?

Well yes, but I have finished that part so I will commit that into here too.

**Pull request merged**